### PR TITLE
Initial security policy draft

### PR DESCRIPTION
**Pull request by erkinalp**
This adds a formal security policy.
@alexanderby @Gusted feedback requested.

### CODE REVIEW

**Comment by erkinalp** [comment]
This adds a formal security policy.

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+| Version   | Supported          |
+| --------- | ------------------ |
+| 4.9.16    | :white_check_mark: |
+| < 4.9.16  | :x:                |
+
+## Reporting a Vulnerability
+
+In case there is a vulnerability left behind, immediately notify
+to <darkreaderapp@gmail.com>; unless you can fix it yourself,
```
Comment (Gusted) [inline SECURITY.md:19]: > unless you can fix it yourself,

So that everyone can see the PR and abuse the bug right away?

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Security Policy
+
+DarkReader is known to have no intentional security vulnerabilities.
```
Comment (Gusted) [inline SECURITY.md:3]: `Dark Reader` instead of `DarkReader`

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+If reported, vulnerabilites will be fixed in the shortest time possible,
```
Comment (Gusted) [inline SECURITY.md:4]: `shortest time and highest priority as possible`

**Review by Gusted — CHANGES_REQUESTED** [review]
Not sure how we even will be vulnerable, as it's a simple extension that only can be modified internal settings trough the UI that always have checks in place. However the API could be, but that's rather that it will load images that aren't needed in the first place. 🤷🏽‍♀️

File: SECURITY.md (hunk 1 above)
Comment (erkinalp) [reply SECURITY.md:19]: I left that bit intentionally, to enable a copyleft relicense, which require the changes the published as soon as possible.

File: SECURITY.md (hunk 3 above)
Comment (erkinalp) [reply SECURITY.md:4]: Fixing priority is deliberately left out as it would imply a warranty policy.

**Review by alexanderby — APPROVED** [review]

**Review by Gusted — APPROVED** [review]

**Pull request merged**